dali/python/nvidia/dali/types.py

```diff
@@ -573,11 +578,11 @@ def Constant(value, dtype=None, shape=None, layout=None, device=None, **kwargs):
     """
 
     if device is not None or \
-        (_is_compatible_array_type(value) and not _is_true_scalar(value)) or \
-        isinstance(value, (list, tuple)) or \
-        not _is_scalar_shape(shape) or \
-        kwargs or \
-        layout is not None:
+       (_is_compatible_array_type(value) and not _is_true_scalar(value)) or \
+       isinstance(value, (list, tuple)) or \
+       not _is_scalar_shape(shape) or \
+       kwargs or \
+       layout is not None:
```

Comment:
```suggestion
    if (device is not None
        or (_is_compatible_array_type(value) and not _is_true_scalar(value))
        or isinstance(value, (list, tuple))
        or not _is_scalar_shape(shape)
        or kwargs
        or layout is not None):
```

Reply:
Your suggestion will not pass the linter. 

I refactored it to be something similar with extra indentations.